src/cow-react/modules/limitOrders/state/partiallyFillableOverride.ts

```diff
@@ -0,0 +1,9 @@
+import { atom, SetStateAction } from 'jotai'
+
+export type PartiallyFillableOverrideType = boolean | undefined
+export type PartiallyFillableOverrideDispatcherType = [
+  PartiallyFillableOverrideType,
+  (update?: SetStateAction<PartiallyFillableOverrideType>) => void
+]
+
+export const partiallyFillableOverrideAtom = atom<PartiallyFillableOverrideType>(undefined)
```

Comment:
What was the reason to add global state for this? 

In principle is just data part of the modal, right?

Reply:
The override needs to be accessible and settable in the dropdown, both for expert (in the form itself) and regular (in the review modal) modes.
The value needs to be accessible when the order is placed.

To avoid too much prop drilling, I used a jotai atom.
What's the alternative you suggest? A local context?

Reply:
Yes, local state, but if it's used in 2 places I guess is justified probably